replica/database.hh

```diff
@@ -396,13 +396,16 @@ public:
         cache_temperature rate;
         lowres_clock::time_point last_updated;
     };
+
+    void deregister_metrics();
```

Comment:
I don't really like the fact that this method is public, because `set_metrics` is private.
On a side note, if this function is to stay, I'll move it to the other public section, where all other functions are listed.